.github/workflows/run-tests.yml

```diff
@@ -19,6 +19,16 @@ jobs:
         run: |
           pip install -r requirements/production.txt
           pip install -r requirements/test.txt
+      - name: Set up Node
```

Comment:
I'll be replacing this setup with running tests inside the Docker container soon